webui/react/src/pages/ExperimentDetails/ExperimentVisualization/LearningCurve.tsx

```diff
@@ -74,12 +77,58 @@ const LearningCurve: React.FC<Props> = ({
   const hasTrials = trialHps.length !== 0;
   const isExperimentTerminal = terminalRunStates.has(experiment.state as RunState);
 
-  const hyperparameters = useMemo(() => {
-    return fullHParams.reduce((acc, key) => {
-      acc[key] = experiment.hyperparameters[key];
+  const get_custom_search_varying_hps = (trialHps: TrialHParams[]) => {
```

Comment:
**nit chore:** we use camel case for function names - `getCustomSearchVaryingHps`